src/main/java/edu/cornell/kfs/concur/ConcurUtils.java

```diff
@@ -65,4 +68,21 @@ public static boolean isConcurReportStatusAwaitingExternalValidation(String stat
         return ConcurConstants.EXPENSE_AWAITING_EXTERNAL_VALIDATION_STATUS_CODE.equalsIgnoreCase(statusCode) || ConcurConstants.REQUEST_AWAITING_EXTERNAL_VALIDATION_STATUS_CODE.equalsIgnoreCase(statusCode);
     }
 
+    /**
```

Comment:
I think we should eliminate this comment and rename the method to something more general like generateBase64EncodedString